examples/howto/plot_hnn_animation.py

```diff
@@ -0,0 +1,67 @@
+"""
+================================
+06. Animating HNN simulations
+================================
+
+This example demonstrates how to animate HNN simulations
+"""
+
+# Author: Nick Tolley <nicholas_tolley@brown.edu>
+
+
+###############################################################################
+# First, we'll import the necessary modules for instantiating a network and
+# running a simulation that we would like to animate.
+import os.path as op
+import hnn_core
+from hnn_core import jones_2009_model, simulate_dipole, read_params
+from hnn_core.network_models import add_erp_drives_to_jones_model
+
+###############################################################################
+# We begin by instantiating the network. For this example, we will reduce the
+# number of cells in the network to speed up the simulations.
+hnn_core_root = op.dirname(hnn_core.__file__)
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+params.update({'N_pyr_x': 3, 'N_pyr_y': 3})
```

Comment:
this won't work with the `hdf5` format. We'll need an API to do this.

Reply:
are you ok to leave this for now and replace when we push #651 over the finish line?

Reply:
I think we can now remove the reference to `default.json` and avoiding using/modifying the params dict here.